src/features/query-bar/components/filter/Condition.js

```diff
@@ -89,15 +89,15 @@ const Condition = ({
                 break;
         }
 
-        update(index, condition, displayText, displayTextType, fieldId, value, keyType, type);
+        update(index, condition, displayText, displayTextType, id, value, keyType, type);
```

Comment:
shouldn't "update" be named "onXYZChange"?

Reply:
Yes, I can change it to be:

handleFieldChange / onFieldChange